ndex: qca-wifi-g/cmn_dev/umac/dfs/core/src/misc/dfs_process_radar_found_ind.c
===================================================================
Index: qca-wifi-g/cmn_dev/umac/dfs/core/src/misc/dfs_process_radar_found_ind.c
===================================================================
--- qca-wifi-g.orig/cmn_dev/umac/dfs/core/src/misc/dfs_process_radar_found_ind.c
+++ qca-wifi-g/cmn_dev/umac/dfs/core/src/misc/dfs_process_radar_found_ind.c
@@ -1169,7 +1169,7 @@ QDF_STATUS dfs_process_radar_ind(struct
 			 radarfound_freq, dfs_curchan->dfs_ch_ieee,
 			 dfs_curchan->dfs_ch_freq);
 
-	utils_dfs_deliver_event(dfs->dfs_pdev_obj, radarfound_freq,
+	utils_dfs_deliver_event(dfs->dfs_pdev_obj, dfs_curchan->dfs_ch_freq,
 				WLAN_EV_RADAR_DETECTED);
 
 	if (!dfs->dfs_use_nol) {
